### PR TITLE
[PCM-52] Add initial integration tests for Video Troubleshooting.

### PR DESCRIPTION
**Pull request by jbelmont**
I need to figure out why phantomjs can't seem to find dom elements for integration test. It passes in chrome but not in phantomjs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "build": "ember build",
     "start": "ember server",
     "coverage": "COVERAGE=true ember test",
-    "test": "node_modules/.bin/semistandard blueprints/**/*.js tests/**/*.js addon/**/*.js app/**/*.js && npm run coverage",
+    "test": "semistandard blueprints/**/*.js tests/**/*.js addon/**/*.js app/**/*.js && npm run coverage",
```
Comment (jensengar) [inline package.json]: lets use local libraries, not global.

File: tests/integration/components/webrtc-troubleshooter/component-test.js [hunk 2]
```diff
@@ -1,19 +1,94 @@
...
 
+const intl = Ember.Service.extend({
+  exists () {
+    return true;
+  },
+  lookup () {
+    return this.t(...arguments);
+  },
+  t (key) {
+    if (key === 'chat.presence.outOfOffice') {
```
Comment (jensengar) [inline tests/integration/components/webrtc-troubleshooter/component-test.js]: why do we care about this?

File: package.json (hunk 1 above)
Comment (jbelmont) [inline package.json]: this is using local libraries, it is unnecessary to to preface with node_modules/.bin it already does that

File: tests/integration/components/webrtc-troubleshooter/component-test.js (hunk 2 above)
Comment (jbelmont) [inline tests/integration/components/webrtc-troubleshooter/component-test.js]: this line is pointless only need to mockout t function so I will remove this condition

File: tests/dummy/translations/en-us.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "webrtcTroubleshoot": {
```
Comment (xdumaine) [inline tests/dummy/translations/en-us.json]: Is there a way to have the dummy app load translations from the addon instead of duplicating strings?

Comment (jbelmont) [inline tests/dummy/translations/en-us.json]: I haven't investigated it this like I should, but I can now

**Comment by jbelmont** [comment]
@xdumaine I added the translation file to resolve errors in phantomjs but am not seeing them when running in headless chrome so I removed the duplication here. Although I am still looking at if there is some option at ember-cli-build.js that I can add to include intl assets

**Pull request merged**